event-sourcing-introduction.md

```diff
@@ -0,0 +1,232 @@
+# Event-Sourcing: A Different Way to Model State
+
+Event-Sourcing is an architectural pattern where we store the history of state changes as a sequence of events, rather than just storing the current state. Let's explore this concept using a relatable example: growing marijuana plants.
+
+## The Scenario
+
+I have a marijuana plant. My grandma has another marijuana plant. Mine is starving and struggling to grow, while the one my grandma tends to is flourishing. Why? Let's see how Event-Sourcing can help us understand the difference.
+
+## CRUD: The Traditional Approach
```

Comment:
CRUD does not need detailed examples. 2 lines of a database table would be enough to visualize it.
Then add another event and show the updated row.